pkgs/by-name/ne/netmaker/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, installShellFiles
+, nix-update-script
+, ...
+}:
+buildGoModule rec {
+  pname = "netmaker";
+
+  passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch=develop" ]; };
+  version = "unstable-2024-01-24";
+
+  src = fetchFromGitHub {
+    owner = "nazarewk";
+    repo = "netmaker";
+    rev = "c2f7a40024ab087ade019e82f2d6f797a894f282";
+    hash = "sha256-P19fGDMFKjiR5LP1kQitTVMLKYiPs0z3ICZaIFfgRzQ=";
+    postFetch = ''
+      # $out/pro holds propertiary licensed code and main_ee.go uses it with correct build flag.
+      rm -r $out/pro
+      rm $out/main_ee.go
+    '';
+  };
+  vendorHash = "sha256-t7g6Tozq/QLq0/5bpXNDCJrOPTjMlvcDUaD6EGqII3Y=";
+
+  subPackages = [ "." "cli" ];
+  CGO_ENABLED = true;
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    mv $out/bin/{cli,nmctl}
+
+    installShellCompletion --cmd nmctl \
+      --bash <($out/bin/nmctl completion bash) \
+      --fish <($out/bin/nmctl completion fish) \
+      --zsh <($out/bin/nmctl completion zsh)
+  '';
+
+  meta = {
+    description = "WireGuard automation from homelab to enterprise - Community Edition";
+    homepage = "https://netmaker.io";
+    changelog = "https://github.com/gravitl/netmaker/-/releases/v${version}";
+    license = lib.licenses.asl20;
```

Comment:
Pro version of netmaker is licensed under `The Netmaker Enterprise license` not `Apache-2.0`

Reply:
this one is not a pro version

Reply:
True I'm sorry , my oversight

Reply:
Then we should also drop the isPro argument

Reply:
That `isPro` is today's addition, I am building 3 flavors of the package through overrides:
- `netmaker` - has only the server binary (`.` subpackage)
- `nmctl` - has only `nmctl` binary (`cli` subpackage)
- `netmaker-pro` - has only the server binary and is licensed as `unfree` (`/pro` subdirectory is not used in `nmctl` at all)